.gitignore

```diff
@@ -2,3 +2,5 @@
 
 .tox/
 .idea/
+
+manage.py
```

Comment:
i just need it to create migrations, should i add it ? it's copied from django source